ui_framework/components/tool_bar/tool_bar_search_box.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, mount } from 'enzyme';
+import sinon from 'sinon';
+import { commonHtmlProps } from '../../test/common_html_props';
+
+import {
+  KuiToolBarSearchBox,
+} from './tool_bar_search_box';
+
+describe('KuiToolBarSearchBox', () => {
```

Comment:
I don't have a strong opinion one way or the other regarding the nesting of describes.  Perhaps I slightly favor @kjbekkelund's approach because it's fewer requirements and I don't think the forced structure really gets us that much, but I'd be fine either way.

I also don't have a strong opinion on whether we should split apart testing all the basic props.  Again perhaps I favor @kjbekkelund's approach a bit more because I think I don't think the added requirements really get us that much.

If we do decide to split apart all the basic props though, I *do* have a strong opinion on how that should be done, and if that's the case then I think we should re-introduce the helper functions, that were removed after the comments above (https://github.com/elastic/kibana/pull/10821#pullrequestreview-31386111).

If we want to encourage developers to add tests, they should be quick and easy to write, otherwise dev's won't do it, and I think that it would be painful to have to rewrite these same tests over and over and over again for every component.  Also, I don't think writing them out every time adds to clarity.  They are more noise than signal because they would be duplicated everywhere.